File: api/src/main/java/com/datastrato/gravitino/file/ClientType.java
/*
 * Copyright 2024 Datastrato Pvt Ltd.
 * This software is licensed under the Apache License version 2.
 */
package com.datastrato.gravitino.file;

/** An enum class containing fileset data operations client type that supported. */
public enum ClientType {
  HADOOP_GVFS,
  PYTHON_GVFS,
  UNKNOWN;
}

File: clients/client-python/gravitino/api/client_type.py
"""
Copyright 2024 Datastrato Pvt Ltd.
This software is licensed under the Apache License version 2.
"""

from enum import Enum


class ClientType(Enum):
    """An enum class containing fileset data operations client type that supported."""

    PYTHON_GVFS = "python_gvfs"
    UNKNOWN = "unknown"
